### PR TITLE
Add icons

### PR DESCRIPTION
**Pull request by firehawk89**
I've set up colors and fonts config for the messenger. However, I didn't find out best practices of using svg icons with Next.js, so I've just used an approach of using a unique component per icon.

Closes https://github.com/dev-KPI/messenger-frontend/issues/47

### CODE REVIEW

File: src/components/UI/Icons/IconChange.tsx [hunk 1]
```diff
@@ -0,0 +1,29 @@
+import { FC } from 'react'
+
+type IconChangeProps = {
```
Comment (DemianParkhomenko) [inline src/components/UI/Icons/IconChange.tsx:3]: Must be shared type between different icons.

File: src/components/UI/Icons/IconChange.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  height?: number
+  width?: number
+}
+
+const IconChange: FC<IconChangeProps> = ({ className, height, width }) => {
+  return (
+    <svg
+      className={className}
+      fill="none"
+      height={height || 24}
```
Comment (DemianParkhomenko) [inline src/components/UI/Icons/IconChange.tsx:14]: Magic number

File: src/components/UI/Icons/IconChange.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+      viewBox="0 0 24 24"
+      width={width || 24}
```
Comment (DemianParkhomenko) [inline src/components/UI/Icons/IconChange.tsx:16]: Magic number

File: tailwind.config.ts [hunk 4]
```diff
@@ -16,24 +16,30 @@ const config: Config = {
...
-        'base-gray-9': 'rgba(99, 99, 102, 1)',
-        'base-white': 'rgba(255, 255, 255, 1)',
-        'bright-blue': 'rgba(53, 113, 207, 1)',
-        'bright-indigo': 'rgba(71, 112, 200, 1)',
-        'bright-orange': 'rgba(255, 149, 0, 1)',
-        'bright-purple': 'rgba(175, 82, 222, 1)',
-        'bright-red': 'rgba(255, 59, 48, 1)',
-        'bright-teal': 'rgba(92, 193, 250, 1)',
-        'bright-yellow': 'rgba(255, 204, 0, 1)',
+        base: {
```
Comment (DemianParkhomenko) [inline tailwind.config.ts:19]: 👍

**Review by DemianParkhomenko — APPROVED** [review]
Some fixes and you can merge it

File: src/components/UI/Icons/IconChange.tsx (hunk 2 above)
Comment (firehawk89) [reply src/components/UI/Icons/IconChange.tsx]: Fixed by using enum

File: src/components/UI/Icons/IconChange.tsx (hunk 3 above)
Comment (firehawk89) [reply src/components/UI/Icons/IconChange.tsx]: Fixed by using enum

**Pull request merged**